### DISCUSSION TITLE
Enable Knightly for repos

### DISCUSSION THREAD

**Discussion by alexanderniebuhr** [windicss/windicss#88] [Q&A] [answered]
Could we setup Knightly  https://github.com/knightlyjs/knightly for windicss?
This project and all the libs will get much tracking soon. This is going to ger promoted so often, as it is exceptionally good. However, many things need to be tested early before even merged and released, so would be a good fit for the pr's to enable Knightly I guess, @antfu would be the right person to ask how it works, i guess.

@voorjaar what do you think?

Libs I think might be happy to have this:
https://github.com/windicss/windicss
https://github.com/windicss/vite-plugin-windicss
https://github.com/windicss/vue-windicss-preprocess
https://github.com/windicss/svelte-windicss-preprocess

**Comment by voorjaar** [comment]
Of course. Like the idea. And would you like to join the [dicord](https://discord.gg/BM3JwkdQ) group?

**Comment by alexanderniebuhr** [comment]
@voorjaar should I submit a request for the project and link it here?

**Reply by voorjaar** [reply]
I think I should discuss this with antfu, he is sleeping, please wait a moment.

**Reply by alexanderniebuhr** [reply]
yes sure. No pressure, just wanted to bring this idea up

**Comment by antfu** [comment, marked as answer]
Sure, why not. I will set up it soon.
